product_docs/docs/bdr/4/overview.mdx

```diff
@@ -111,7 +111,7 @@ BDR places a limit that at most 10 databases in any one PostgreSQL instance
 can be BDR nodes across different BDR node groups. However, BDR works best if
 you use only one BDR database per PostgreSQL instance.
 
-The minimum recommended number of nodes in a BDR cluster is three to ensure the consensus mechanism works properly. Consensus is a challenge in fault-tolerant distributed systems. Consensus involves multiple nodes in a cluster agreeing on values. BDR uses a [Raft](https://raft.github.io/)-based consensus mechanism. The mechanism makes progress towards a decision as long as a majority of nodes is available. For example, if you only have two nodes in your cluster, the consensus mechanism stops working if one of the nodes stops working. 
+The minimum recommended number of nodes in a BDR cluster is three because with two nodes the consensus stops working if one of the node stops working. For more information about the consensus mechanism used by EDB Postgres Distributed, see [Architectural details](/pgd/latest/architectures/#architecture-details).
```

Comment:
```suggestion
The minimum recommended number of nodes in a BDR cluster is three to provide fault tolerance for BDR's consensus mechanism. With just two nodes, consensus would fail if one of the nodes was unresponsive. Consensus is required for some BDR operations such as distributed sequence generation. For more information about the consensus mechanism used by EDB Postgres Distributed, see [Architectural details](/pgd/latest/architectures/#architecture-details).
```